src/contrasts.jl

```diff
@@ -224,6 +266,17 @@ regression model, dummy coding leads to an intercept that is the mean of the
 dependent variable for base level.
 
 Also known as "treatment coding" (`contr.treatment` in R) or "one-hot encoding".
```

Comment:
I don't think there's an additional coefficient coefficient to report for forward/reverse difference coding; anyway, coefficient names should include all original levels (as part of the representation of differences). For Helmert coding, we cannot compare the last level with subsequent levels, so there's nothing to compute either; though it could be useful to have just the name of the last level with a `NaN` coefficient, so that you know the name of the last level. I really can't say, as I've only used dummy and effects coding in practice.

We don't need to print this by default. But it would be a very useful information to generate publication-level regression tables, where you always need to give the full information about all levels. These additional coefficients could be printed in a different way from others: for example, the base level for dummy coding could be printed with `0` next to the variable name, after "Reference:".